nativeruntime/build.gradle

```diff
@@ -41,16 +41,18 @@ task configureICU {
     if (!file("$projectDir/external/icu/icu4c/source").exists()) {
       throw new GradleException("ICU submodule not detected. Please run `git submodule update --init`")
     }
-    exec {
-      workingDir "$projectDir/external/icu/icu4c/source"
-      if (os.contains("linux")) {
-        environment "CFLAGS", "-fPIC"
-        environment "CXXFLAGS", "-fPIC"
-        commandLine './runConfigureICU', 'Linux', '--enable-static', '--disable-shared'
-      } else if (os.contains("mac")) {
-        commandLine './runConfigureICU', 'MacOSX', '--enable-static', '--disable-shared'
-      } else {
-        println("Skipping the nativeruntime build for OS '${System.getProperty("os.name")}'")
+    if (!file("$projectDir/external/icu/icu4c/source/Makefile").exists()) {
+      exec {
+        workingDir "$projectDir/external/icu/icu4c/source"
+        if (os.contains("linux")) {
+          environment "CFLAGS", "-fPIC"
+          environment "CXXFLAGS", "-fPIC"
+          commandLine './runConfigureICU', 'Linux', '--enable-static', '--disable-shared'
+        } else if (os.contains("mac")) {
+          commandLine './runConfigureICU', 'MacOSX', '--enable-static', '--disable-shared'
+        } else {
+          println("Skipping the nativeruntime build for OS '${System.getProperty("os.name")}'")
```

Comment:
"Skipping the ICU configuration for OS ..."?

Reply:
Done, just had a simple message `ICU Makefile detected, skipping ICU configure`